### PR TITLE
Created login app and added user account model

### PR DESCRIPTION
**Pull request by mayasa3**
Added user account model to models.py

Related issues #23, #26

### CODE REVIEW

File: login/models.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+class Gender(models.TextChoices):
+    MALE = 'M', 'Male'
+    FEMALE = 'F', 'Female'
+    NONBINARY = 'NB', 'Nonbinary'
+    UNSPECIFIED = 'U', 'Unspecified'
+
+
+class User_account(models.Model):
```
Comment (bardielle) [inline login/models.py]: User_account -> UserAccount

File: login/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# from django.test import TestCase
+
+# Create your tests here.
```
Comment (bardielle) [inline login/tests.py:3]: write the relevant issue for those tests

File: login/views.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# from django.shortcuts import render
+
+# Create your views here.
```
Comment (bardielle) [inline login/views.py:3]: write the relevant issue for those views.

File: login/tests.py (hunk 2 above)
Comment (advapi) [reply login/tests.py:3]: does it need to be in a separate issue or is it ok to edit the issue was specified in the PR content?

Comment (bardielle) [reply login/tests.py:3]: I think that it needs to be a separate issue

File: login/models.py (hunk 1 above)
Comment (mayasa3) [reply login/models.py]: Done

File: joboard/settings.py [hunk 4]
```diff
@@ -38,6 +38,7 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'joboard_home.apps.JoboardHomeConfig',
+    'login'
```
Comment (bardielle) [inline joboard/settings.py:41]: Please split that commit from the commit in the master (it seems that you edited the last master commit)

Comment (mayasa3) [reply joboard/settings.py:41]: Just making sure I understand - should I split the change from the last commit and move it to the other one or remove the commit from this PR?

Comment (bardielle) [reply joboard/settings.py:41]: You need to split your first commit: 978db81089e37dc4874753b15e0786db51a31e54

Comment (mayasa3) [reply joboard/settings.py:41]: Does it look ok now?

Comment (guy9050) [reply joboard/settings.py:41]: I think yes. 
@bardielle?

Comment (bardielle) [reply joboard/settings.py:41]: Yes:)

**Pull request merged**